R/util_assert.R

```diff
@@ -80,6 +80,37 @@ assert_file_exists <- function(files, name = "File", call = NULL, arg = NULL) {
 }
 
 
+find_entrypoint_filename <- function(src, reportname,
+                                       suppress_errors = FALSE) {
+  names <- c(sprintf("%s.R", reportname), "orderly.R")
+  files_exist <- file.exists(file.path(src, names))
+  if (sum(files_exist) > 1 && !suppress_errors) {
+    cli::cli_abort(
+      paste("Please only create {names[[1]]} file, orderly.R",
+            "has been deprecated")
+    )
+  } else if (sum(files_exist) == 0 && !suppress_errors) {
```

Comment:
Our normal style here is not to have an else clause after an if that throws.  So

```
if (whatever) {
  throw
}
otherwise
```

not

```
if (whatever) {
  throw
} else {
  otherwise
}
```

Reply:
yep done!